### ISSUE TITLE
No name property announced for checkbox present beside the "Select Subscription" control.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/vscode#143855]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#VisualStudioCode-Win32-Apr2021;#Visual Studio Code Client;#A11ySev2;#WCAG4.1.2;#DesktopApp;#Win10;#NVDA;#JAWS

### Environment Details:
Visual Studio Code
Azure Function

### Steps to Reproduce:

1. Launch the Visual studio code.
2. TAB to on "Extension" button and search for "Azure Functions" and select it using ENTER key.
3. "Azure Function" screen should open.
4. TAB to "Azure" button and hit ENTER key.
5. Start NVDA and TAB to "Filter" button present in Subscriptions section.
6. TAB to checkbox present beside the "Select Subscription" control.
7. Observe the announcement by screen reader.

### Actual:
No name property announced by NVDA for checkbox present beside the "Select Subscription" control.
This issue is also observed with JAWS.

### Expected:
The NVDA should announce the appropriate name property for checkbox present beside the "Select Subscription" control such as "Select All subscription" checkbox.

### User Impact:
Screen reader users won't be able to understand the purpose of the control.

Please find the attachment for reference:
![image](https://user-images.githubusercontent.com/54656883/155467162-2dfdd922-d4ac-44c4-818f-7a796ca1d6cc.png)

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!
